### DISCUSSION TITLE
model.info(...)调用时，无法正常获取到国际化的 context

### DISCUSSION THREAD

**Discussion by zkylearner** [umijs/umi#11455] [Q&A]
项目开启了国际化功能，部分场景需要使用 model.info({context: <App />, ...}) 这种静态方法调用。传入的 App 组件 调用 useIntl()时报错，无法正常获取到 context。

报错信息

Error: [React Intl] Could not find required `intl` object. <IntlProvider> needs to exist in the component ancestry.

组件本身支持传入context，但是没有在 umi 文档中找到获取对应国际化的 context， 请教下这种情况怎么处理

环境信息

umi 4.0.72
antd 4.24.12

**Comment by fz6m** [comment]
antd 4 应该没有办法，这个问题在 antd 5 里是提供了一个 [`App`](https://ant.design/components/app-cn) 作用域来保持上下文。

考虑自己在全局 layout 上维护一个公共的 modal ，方法提供下去，下面都用这一个 modal 吧。
